Arithmetic.pp

```diff
@@ -65,7 +66,10 @@
     ternary() ( ::times:: #multiplication expression() )?
 
 ternary:
-    term() ( ::div:: #division expression() )?
+    quaternary() ( ::div:: #division expression() )?
+
+quaternary:
+    term() ( ::pow:: #power expression() )?
```

Comment:
According to many sources found on Google (!!), ```^```is lower than multiplication and division operators.